README.md

```diff
@@ -10,18 +10,50 @@ A ruby REST client that lets you feel like in heaven when consuming APIs.
 
         gem 'rest-in-peace'
 
-2. Choose which http adapter you want to use
+2. Choose which HTTP client you want to use
 
         gem 'faraday'
+        
+3. Choose which HTTP adapter you want to use
+
+        gem 'excon'
 
 ## Usage
 
 ### HTTP Client Library
 
-There is no dependency on a specific HTTP client library but the client has been tested with [Faraday](https://github.com/lostisland/faraday) only. You can use any other client library as long as it has the same API as Faraday.
+There is no dependency on a specific HTTP client library but the client has been tested with combination of [Faraday](https://github.com/lostisland/faraday) and [Excon](https://github.com/excon/excon) only. You can use any other client library as long as it has the same API as Faraday.
```

Comment:
IMHO this isn't totally correct. The only dependency is faraday, what kind of adapter is being used in Faraday doesn't matter for RiP. Basically RiP supports every adapter that Faraday supports.